### ISSUE TITLE
`pyreverse` should ignore class level attribute declarations

### ISSUE THREAD

**Issue by DudeNr33** [pylint-dev/pylint#8189]
### Bug description

If an attribute is only declared on class level (with type annotation), but not assigned to a value, it is currently displayed twice in `pyreverse` diagrams:

```python
class ExampleClass():
    example1: int
    example2: int

    def __init__(self):
        self.example1 = 1
        self.example2 = 2
```

```plantuml
@startuml classes
set namespaceSeparator none
class "ExampleClass" as asdf.ExampleClass {
  example1 : int
  example1 : int
  example2 : int
  example2 : int
}
@enduml
```

However it should be ignored, and only the type information should be picked up by `pyreverse`.

Note: if a value is assigned on class level, the attribute is a class attribute and the attributes defined in `__init__` are instance attributes.
Both exist independently from one another, and the class attribute should be shown as "static attribute" with an underline.
See also the discussion in [this](https://github.com/PyCQA/pylint/issues/8046#issuecomment-1417167700) issue.

### Command used

```shell
pyreverse a.py
```

### Pylint output

```shell
See above
```

### Expected behavior

Only show the attribute once.

### Pylint version

```shell
pylint 2.16.0-dev
astroid 2.12.13
Python 3.10.8 (main, Nov  1 2022, 14:18:21) [GCC 12.2.0]
```
